sdk/search/search-documents/test/internal/synonymMap.browser.spec.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+
+describe("synonymmap", () => {
+  it.skip("create synonymmap from file", async function() {
```

Comment:
I have added .skip in the browser.spec.ts file. Is it acceptable?

Reply:
We usually just assert that it throws

Reply:
@xirzec Thanks. Updated in the latest commit